.github/workflows/integration.yml

```diff
@@ -292,6 +291,7 @@ jobs:
             .github/workflows/integration.yml
             .proxy-version
             viz/**
+            test/integration/viz/**
```

Comment:
We should probably run the integration tests when the tests themselves change. Thought I'd do it now (and bump Go to v1.18 while I'm at it) instead of opening a new PR.